work/v1/0000_00_work.open-cluster-management.io_manifestworks.crd.yaml

```diff
@@ -43,14 +43,18 @@ spec:
             workload:
               description: Workload represents the manifest workload to be deployed
                 on spoke cluster
-              type: object
-              properties:
-                manifests:
-                  description: Manifests represents a list of kuberenetes resources
-                    to be deployed on the spoke cluster.
-                  type: array
-                  items:
+              type: array
+              items:
+                description: ManifestsTemplate represents the manifest workload to
+                  be deployed on spoke cluster
+                type: object
+                properties:
+                  manifest:
+                    description: Manifest represents a resource to be deployed on
+                      the spoke
                     type: object
+                    x-kubernetes-preserve-unknown-fields: true
```

Comment:
fyi, this means you can put in any object type (pods, services, builds, anything...). If you know this type more concretely, you could also specify the schema here instead of `x-kubernetes-preserve-unknown-fields: true`.

Reply:
> fyi, this means you can put in any object type (pods, services, builds, anything...). If you know this type more concretely, you could also specify the schema here instead of `x-kubernetes-preserve-unknown-fields: true`.

it really is an anything field.